### PR TITLE
Control-flow: Loops & Answers

### PR DESCRIPTION
**Pull request by lindbrook**
In Line 38, "`perform_action` is called once for each item in `vector`, rebinding the value of `item` each time.", what do you mean by "rebinding"? incrementing?

### CODE REVIEW

**Comment by hadley** [comment]
I mean that it's like `item <- 1; item <- 2; item <- 3`

File: Control-flow.Rmd [hunk 1]
```diff
@@ -257,8 +257,8 @@ i
 \indexc{break}
 There are two ways to terminate a for loop early:
 
-* `next` terminates the current iteration and advances to the next.
-* `break` exits the entire for loop.
+* `next` stops the current iteration and advances to the next one.
```
Comment (hadley) [inline Control-flow.Rmd]: `one` feels weird to me here

Comment (lindbrook) [reply Control-flow.Rmd]: How about: `next` stops the current iteration and moves on to the next iteration.

Comment (lindbrook) [reply Control-flow.Rmd]: "rebinding" -> "updating"?

Comment (hadley) [reply Control-flow.Rmd]: Even iteration feels redundant to me. Updating sounds good

Comment (lindbrook) [reply Control-flow.Rmd]: 'next' exits the current loop.
'break' exits the entire for loop.

Comment (hadley) [reply Control-flow.Rmd]: next exits the current iteration
break exits the entire for loop
?

**Comment by hadley** [comment]
Thanks!

**Pull request merged**